scripts/sdk_process/patch_models.py

```diff
@@ -231,8 +231,9 @@ def find_autorest_generated_folder(module_prefix="azure.mgmt"):
         'azure.mgmt.compute',
         'azure.mgmt.network',
         'azure.mgmt.monitor',
-        'azure-mgmt-loganalytics'
         'azure.mgmt.rdbms'
+        'azure-mgmt-loganalytics',
+        'azure-mgmt.web'
```

Comment:
```suggestion
        'azure.mgmt.loganalytics',
        'azure.mgmt.web'
```

Reply:
@fengzhou-msft Ah! I will .web - log analytics isn't an extension I own; I hope its still ok for me to update this as well?